freeipa/src/main/resources/freeipa-salt/salt/freeipa/init.sls

```diff
@@ -26,4 +26,28 @@ install-freeipa:
         - REALM: {{salt['pillar.get']('freeipa:realm')}}
     - unless: test -f /var/log/freeipa_install-executed
     - require:
-        - file: /opt/salt/scripts/freeipa_install.sh
\ No newline at end of file
+        - file: /opt/salt/scripts/freeipa_install.sh
+
+
+/opt/salt/scripts/ipa_user_management.sh:
+  file.managed:
+    - makedirs: True
+    - user: root
+    - group: root
+    - mode: 700
+    - source: salt://freeipa/scripts/ipa_user_management.sh
+
+add-user:
+  cmd.run:
+    - name: /opt/salt/scripts/ipa_user_management.sh && echo $(date +%Y-%m-%d:%H:%M:%S) >> /var/log/ipa_user_management-executed
+    - env:
+        - ADMINUSER: {{salt['pillar.get']('freeipa:password')}}
+        - PASSWORD: {{salt['pillar.get']('freeipa:domain')}}
+        - USERNAME: {{salt['pillar.get']('freeipa:username')}}
```

Comment:
I think fname, lname, username are not set in pillar right now. https://github.com/hortonworks/cloudbreak/blob/master/freeipa/src/main/java/com/sequenceiq/freeipa/service/FreeIpaInstallService.java#L54-L57